src/Searchbar.vue

```diff
@@ -49,6 +49,32 @@ export default {
       return value.replace(new RegExp(`(${phrase})`, 'gi'), '<mark>$1</mark>');
     },
   },
+  methods: {
+    down() {
+      if (this.current < this.items.length - 1) {
+        this.current += 1;
+        this.scrollListView();
+      }
+    },
+    up() {
+      if (this.current > 0) {
+        this.current -= 1;
+        this.scrollListView();
+      }
+    },
+    scrollListView() {
+      const { dropdown } = this.$els;
+      const curEntry = dropdown.children[this.current];
+      const upperBound = dropdown.scrollTop;
+      const lowerBound = upperBound + dropdown.clientHeight;
+      const currentEntryOffsetBottom = curEntry.offsetTop + curEntry.offsetHeight;
+      if (curEntry.offsetTop < upperBound) {
+        dropdown.scrollTop = curEntry.offsetTop;
+      } else if (currentEntryOffsetBottom > lowerBound) {
+        dropdown.scrollTop = (upperBound + currentEntryOffsetBottom) - lowerBound;
```

Comment:
This can be simplified to just `dropdown.scrollTop = currentEntryOffsetBottom - dropdown.clientHeight;`